### DISCUSSION TITLE
Here is how to handle merge conflicts with keyboard only

### DISCUSSION THREAD

**Discussion by keithschacht** [jesseduffield/lazygit#3674] [General]
I love the support for keyboard shortcuts in lazygit, but one thing it took me forever to figure out was how to handle merge conflicts solely with the keyboard.

When I get a merge conflict, I can step through the conflicted files (in pane [2]). And after highlighting a file it shows me the conflicts for that file in the main panel. And I can scroll that main panel with Shift+J / K but I couldn't figure out how to focus.

The answer is: press `return` on the file you have selected. That focuses the main panel. Esc unfocuses and takes you back to the file list.

It took me a bunch of trial and error. I even read through most of the keybindings.
https://github.com/jesseduffield/lazygit/blob/master/docs/keybindings/Keybindings_en.md

I'm sharing here to save some future person time...

**Comment by monoira** [comment]
Thanks man.
This really helped me figuring out how to fix merge conflicts since documentation is iffy and so is creator's 12 min super fast video on yt.

**Reply by keithschacht** [reply]
Ha, I forgot I posted this! At least twice in the last month I’ve tried to do merge conflict and couldn’t remember how to do it. So my past self just gave my current self a refresher. :)
